src/nets/e2e_asr_attctc_th.py

```diff
@@ -26,10 +26,15 @@
 from e2e_asr_common import end_detect
 from e2e_asr_common import label_smoothing_dist
 
+
+torch_is_old = torch.__version__.startswith("0.3.")
+
 CTC_LOSS_THRESHOLD = 10000
 CTC_SCORING_RATIO = 1.5
 MAX_DECODER_OUTPUT = 5
 
+torch_is_old = torch.__version__.startswith("0.3.")
```

Comment:
Do we need to do the same line twice? (`torch_is_old = torch.__version__.startswith("0.3.")`)